kivy/uix/actionbar.py

```diff
@@ -249,6 +249,10 @@ class ActionGroup(ActionItem, Spinner):
        defaults to 'normal'.
     '''
 
+    dropdown_width = NumericProperty(0)
+    '''If non zero, provides the width for the associated DropDown.
```

Comment:
In my opinion, your documentation is too minimal, especially for a new property.
Should be more like:
```python
'''If non zero, provides the width for the associated DropDown.
.. versionadded:: 1.9.0
:attr:`dropdown_width` is an :class:`~kivy.properties.NumericProperty` and defaults to 0.
'''
```